Server/mods/deathmatch/logic/CConsole.cpp

```diff
@@ -119,10 +119,10 @@ bool CConsole::HandleInput(const char* szCommand, CClient* pClient, CClient* pEc
     return false;
 }
 
-void CConsole::AddCommand(FCommandHandler* pHandler, const char* szCommand, bool bRestricted)
+void CConsole::AddCommand(FCommandHandler* pHandler, const char* szCommand, bool bRestricted, string sConsoleHelp)
 {
     // Make a command class and add it to the list
-    CConsoleCommand* pCommand = new CConsoleCommand(pHandler, szCommand, bRestricted);
+    CConsoleCommand* pCommand = new CConsoleCommand(pHandler, szCommand, bRestricted, sConsoleHelp);
     m_Commands.push_back(pCommand);
```

Comment:
imho keep using const char*, not need for string here at all. (if you really want to use one, use std::move). Also, wrong hungarian notation, it should be str not s

Reply:
no need for it, because its an unnecessary copy. (from sConsoleHelp, into  `CConsoleCommand` ctor, and then into a member variable. Thats actually 3 unnecessary copies. To be honest, i would store it as a `const char*` member variable instead, since these are all static texts)

Reply:
Ah I see